incubator/mssql/templates/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+              {
+                  "name": "remove-lost-found",
+                  "image": "busybox:1.25.0",
+                  "command": ["rm", "-fr", "/var/opt/mssql/data/lost+found"],
+                  "volumeMounts": [
+                      {
+                          "name": "data",
+                          "mountPath": "/var/opt/mssql/data"
+                      }
+                  ],
+                  "imagePullPolicy": {{ .Values.imagePullPolicy | quote }}
+              }
+          ]'
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "microsoft/mssql-server-linux:{{ .Values.imageTag }}"
```

Comment:
idk if this has changed, but something to be aware of is (at least at one time) this image was only good for an evaluation period of 100-some days (120?) starting from the date the image was produced-- meaning that this is a ticking time bomb for anyone using it. It will eventually stop functioning unless upgraded with a newer image.